specs/serving/knative-api-specification-1.0.md

```diff
@@ -1173,6 +1173,26 @@ constitutes a request.
    <td>RECOMMENDED
    </td>
   </tr>
+  <tr>
+   <td><code>actualReplicas</code>
+   </td>
+   <td>int
+   </td>
+   <td>Reflects the number of replicas that are running this revision.
```

Comment:
do we need to be clear about whether this is "ready" replicas vs "existing" replicas in any state?

Reply:
I don't think the word `Ready` would have meaning here without more definition. That's why I left the verbiage as `running this revision`

Reply:
What was the original reason for adding these? I have a vague recollection that we added it so that we didn't rely on the Deployment's metadata to maintain Knative's desired state of the world - and then we'd reflect that down into the Deployment. So, I guess if that's all it is and there's no assumption as to state of the underlying "pods" (rather it's just implying that we're "trying" to bring up X of them), is this your thinking too?